tapir/coop/templates/coop/email/resignedmembership_confirmation_body.html

```diff
@@ -0,0 +1,75 @@
+{% extends "core/email_base.html" %}
+{% load i18n %}
+{% load utils %}
+{% block body %}
+    (English below)
+    <p>Liebe*r {% get_display_name_short share_owner %},</p>
+    <p>
+        schade, dass Du uns verlassen möchtest. Hiermit bestätigen wir Dir die Kündigung deiner Mitgliedschaft bei SuperCoop Berlin eG.
+    </p>
+    <p>
+        Wenn Du in deinem Kündigungsschreiben angegeben hast, dass Du deine Anteile an SuperCoop oder ein anderes Mitglied
+        übertragen möchtest, ist deine Kündigung ab sofort gültig.
+    </p>
+    <p>
+        Andernfalls läuft deine Mitgliedschaft bis zum Ablauf der in unserer Satzung angegebenen regulären Kündigungsfrist, d.h.
+        bis zum Abschluss des Geschäftsjahres in drei Jahren. <br> Bitte beachte dazu den in unserer Satzung angegebenen Passus 
+        unter <b>§4 Beendigung der Mitgliedschaft, Übertragung des Geschäftsguthabens, Tod und Ausschluss</b>.<br>
+        Vorsorglich ist an diese Mail die Satzung angehängt. 
+    </p>
+    <p>
+        <b>
+            Da deine Mitgliedschaft im Falle der regulären Kündigungsfrist und damit einhergehenden Auseinandersetzung (siehe Satzung) 
+            über drei Jahre weiterläuft und auch in Tapir weiterhin als "aktiv" eingetragen ist, möchten wir dich an dieser Stelle 
+            darauf hinweisen, dass Du innerhalb der nächsten drei Jahre auch weiterhin Schichten absolvieren kannst und 
+            damit auch dein Einkaufsrecht weiterhin behältst. Solltest Du keine Schichten mehr absolvieren und auch nicht 
+            mehr in der Supercoop einkaufen wollen, möchten wir dich bitten, beim 
+            <a href="mailto: mitglied@supercoop.de">Mitgliederbüro</a> zu beantragen, dass deine Mitgliedschaft
+            auf "investierend" umgestellt wird.
+        </b>
+    </p>
+    <p>
+        Damit wird verhindert, dass dein Schichtstatus ins Minus gerät und Du in Zukunft Emails mit Warnungen von uns erhältst.
+    </p>
+    <p>
+        Wir wünschen Dir alles Gute für die Zukunft und zögere bitte nicht, wenn Du noch weitere Fragen hast, 
+        das <a href="mailto: mitglied@supercoop.de">Mitgliederbüro</a> zu kontaktieren. 
+    <p>
+        Kooperative Grüße,
+    </p>
+    <p>
+        Eugénie, Johanna und Marie für das SuperCoop Team
+    </p>
+    <hr>
+    <p>Dear {% get_display_name_short share_owner %},</p>
+    <p>
+        what a pity that you want to leave us. We hereby confirm the cancellation of your membership with SuperCoop Berlin eG. <br>
+        If you stated in your letter of cancellation that you want to transfer your shares to SuperCoop or another member,
+        your termination is effective immediately.
+    </p>
+    <p>
+        Otherwise, your membership will run until the end of the regular cancellation period specified in our Articles of Association, i.e.
```

Comment:
Since we already have the information which case has been chosen by the member, it would be nice to show only the relevant information to the member.

Reply:
Done. Hopefully correct ;)